install/zowe-install.sh

```diff
@@ -148,6 +148,22 @@ cp $INSTALL_DIR/scripts/ocopyshr.sh $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.sh
 cp $INSTALL_DIR/scripts/ocopyshr.clist $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.clist
 echo "Copying the run-zowe.sh into "$ZOWE_ROOT_DIR/scripts/internal >> $LOG_FILE
 
+# Create the /fingerprint directory in the ZOWE_ROOT_DIR runtime directory,
+# if it exists in the INSTALL_DIR driectory
+if [[ -d $INSTALL_DIR/fingerprint ]]
```

Comment:
Script `zowe-install.sh` is called in different cases.  In this case, its purpose is merely to lay out the structure of the ROOT_DIR runtime, so that the runtime can be hashed.  The resultant hashes are placed in a new directory `fingerprint`.  In order to determine what `zowe-install.sh` should do when it is called, it looks at the `fingerprint` directory.  If it's missing, the script creates it.

An alternative means to determine if it's time to create the hash, is to add a new parameter but that could mean changing all invocations of it.  

So to answer your question, the fingerprint directory won't be in ROOT_DIR until it's created - by this script, indirectly.  

This case of its invocation is just a tentative laying down of ROOT_DIR; the sole purpose is to generate the hashes.  It's not the product and the fingerprint is not placed in ROOT_DIR, which is discarded.  The fingerprint is placed in the directory of files that will be PAXed - only at that point does it become part of the product.